.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/bin/python3.7"
+}
```

Comment:
Please check "git status" before committing, and refrain from using "git commit -a" too much :)